ast3/Python/ast.c

```diff
@@ -2587,6 +2603,11 @@ ast_for_atom_expr(struct compiling *c, const node *n)
     nch = NCH(n);
 
     if (TYPE(CHILD(n, 0)) == AWAIT) {
+        if (c->c_feature_version < 5) {
+            ast_error(c, n,
+                    "Await expressions are only supported in Python 3.5 and greater");
+            return 0;
```

Comment:
Whoops, I think I must have forgotten to change this after copy-pasting. In any case, fixed.